connectivity/lwipstack/include/lwipstack/LWIPStack.h

```diff
@@ -222,6 +225,8 @@ class LWIP : public OnboardNetworkStack, private mbed::NonCopyable<LWIP> {
             void *hw; /**< alternative implementation pointer - used for PPP */
         };
 
+        NetworkInterface *userNetworkInterface;
```

Comment:
`snake_case` is used for other fields, please change it to make it consistent: `user_network_interface`.

Reply:
Done